### PR TITLE
Contact Us page added in router component

### PR DESCRIPTION
**Pull request by sid-khuntwal**

### CODE REVIEW

File: src/components/ContactUs/Contact.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import './Contact.css'
+document.body.style = 'background: #59abe3;';
+
+export default function Contact() {
+  return (
+    <div>
```
Comment (princiya) [inline src/components/ContactUs/Contact.js:7]: you can define classname here and write CSS for the classname `background: #59abe3;`. please don't use `document.body.style`.

File: src/components/ContactUs/Contact.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  display: block;
+  margin: 28px 0 0;
+  font-size: 14px;
+  color: #5a5a5a;
+}
+.sendMessageButton {
+  text-align: left;
+  padding: 8px 12px;
+  margin: 8px 0 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (princiya) [inline src/components/ContactUs/Contact.css:56]: same comment from previous PR. let's not use separate font for every className in Contact.css. Let the contact page derive fonts from the global CSS in place.
please delete all the font-family lines in this file.

File: src/components/ContactUs/Contact.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/* body {
```
Comment (princiya) [inline src/components/ContactUs/Contact.css:1]: just delete these lines instead of commenting them out.

**Comment by princiya** [comment]
good job overall @sid-khuntwal 👏

**Comment by sunithapatel** [comment]
What @princiya said. :) It looks nice!

File: src/components/ContactUs/Contact.css (hunk 2 above)
Comment (sid-khuntwal) [reply src/components/ContactUs/Contact.css:56]: Okay! sure.

**Comment by sid-khuntwal** [comment]
> good job overall @sid-khuntwal 👏

Thanks!! Means a lot 😊

**Comment by sid-khuntwal** [comment]
Thank You @princiya 😊

**Pull request merged**